connectors/slack/template/configure.js

```diff
@@ -16,23 +19,21 @@ module.exports = {
 
             const selfUrl = Sdk.getSelfUrl(ctx);
 
-            const authorizationUrl = `https://slack.com/oauth/v2/authorize?scope=${
-                encodeURIComponent(ctx.configuration.slack_scope || '')
-            }&client_id=${
-                encodeURIComponent(ctx.configuration.slack_client_id)
-            }&redirect_uri=${
-                encodeURIComponent(selfUrl)
-            }&state=${
-                encodeURIComponent(Sdk.serializeState(state))
-            }`
+            const authorizationUrl = `https://slack.com/oauth/v2/authorize?scope=${encodeURIComponent(
```

Comment:
Just for code clarity, create the url parameters as separate variables.  You could even consider using the URL() object.

Reply:
It would have been nice to refactor this, but at this point it would be expensive to retest all the flows